### PR TITLE
fix(INT-339): marketo response handler to handle static list removed and skipped statuses

### PR DESCRIPTION
**Pull request by anantjain45823**
## Description of the change

> We are not handling the `skipped` (code: 1015) and `removed` statuses properly due which for these even if the calls are successful we are returning as error 

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/marketo/util.js [hunk 1]
```diff
@@ -34,29 +36,28 @@ const MARKETO_ABORTABLE_CODES = [
...
+//   '1016',
+//   '1017',
+//   '1018',
+//   '1021',
+//   '1026',
+//   '1027',
+//   '1028',
+//   '1036',
+//   '1049',
+// ];
```
Comment (utsabc) [inline src/v0/destinations/marketo/util.js:60]: Why are we commenting these?

File: src/v0/destinations/marketo/util.js [hunk 2]
```diff
@@ -122,17 +123,27 @@ const marketoApplicationErrorHandler = (marketoResponse, sourceMessage, destinat
...
+          (errorCode) => !ERROR_CODE_TO_PASS.includes(errorCode),
+        );
+        if (!filteredErrorCode) {
+          return;
+        }
+        if (MARKETO_THROTTLED_CODES.includes(filteredErrorCode.code)) {
+          statusCode = 429;
+        } else if (MARKETO_RETRYABLE_CODES.includes(filteredErrorCode.code)) {
+          statusCode = 500;
+        }
```
Comment (utsabc) [inline src/v0/destinations/marketo/util.js:146]: What about abortable codes?

Comment (anantjain45823) [reply src/v0/destinations/marketo/util.js:146]: We are/were taking the error as abortable by default

File: src/v0/destinations/marketo/util.js (hunk 1 above)
Comment (anantjain45823) [reply src/v0/destinations/marketo/util.js:60]: We are by default already converting these to 400 so these are not used anymore but kept them as it is for future ref as there remains some confusion about marketo status codes

**Review by utsabc — APPROVED** [review]

**Review by mihir-4116 — APPROVED** [review]

**Pull request merged**